src/components/AsideBar/AsideBar.jsx

```diff
@@ -20,31 +21,35 @@ const AsideContainer = styled.aside`
     max-width: none;
     gap: 32px;
   }
+  @media (min-width: ${props => props.theme.breakpoints.desktopForMaxMedia}){
+    border-right: 2px solid #E7E5F2;
+    box-shadow: box-shadow: 1px 0 0 0 rgba(255, 255, 255, 0.6);
```

Comment:
wysyłam fix do tego i merge

jeśli nie masz to polecam zainstalować, powinno krzyczeć jeśli takie sytuacje będą
![image](https://github.com/pawelrogowski/goit-wallet-app-03-frontend/assets/110867327/8ae6ee90-8de9-42a2-bb09-d7849ae6a340)